app/models/profile.rb

```diff
@@ -11,11 +11,13 @@ class Profile < ApplicationRecord
   include ProfileRules
 
   SORTABLE_BY = {
-    name: :name,
+    name: Arel.sql('COALESCE(policies.name, profiles.name)'),
     os_minor_version: :os_minor_version,
     score: :score
   }.freeze
 
+  default_scope { left_outer_joins(:policy) }
```

Comment:
I wonder whether this would not cause any issues. Like an ambiguous column select.

Reply:
That only happens when you're using string-based attributes in `ActiveRecord::Base#select`, passing a symbol to this method would always default to `tablename.field`. You can do a full text search for `.select(` and find all of these easily.